app/server/static/js/annotator.vue

```diff
@@ -0,0 +1,168 @@
+<template lang="pug">
+  div(@click="setSelectedRange")
+    span.text-sequence(
+      v-for="r in chunksWithLabel"
+      v-bind:class="{ tag: id2label[r.label].text_color }"
+      v-bind:style="{ \
+        color: id2label[r.label].text_color, \
+        backgroundColor: id2label[r.label].background_color \
+      }"
+    ) {{ [...text].slice(r.start_offset, r.end_offset).join('') }}
+      button.delete.is-small(v-if="id2label[r.label].text_color", v-on:click="removeLabel(r)")
+</template>
+
+<script>
+export default {
+  props: {
+    labels: {
+      type: Array, // [{id: Integer, color: String, text: String}]
+      default: () => [],
+    },
+    text: {
+      type: String,
+      default: '',
+    },
+    entityPositions: {
+      type: Array, // [{'startOffset': 10, 'endOffset': 15, 'label_id': 1}]
+      default: () => [],
+    },
+  },
+
+  data: () => ({
+    startOffset: 0,
+    endOffset: 0,
+  }),
+
+  computed: {
+    sortedEntityPositions() {
+      /* eslint-disable vue/no-side-effects-in-computed-properties */
+      this.entityPositions = this.entityPositions.sort((a, b) => a.start_offset - b.start_offset);
+      return this.entityPositions;
+      /* eslint-enable vue/no-side-effects-in-computed-properties */
+    },
+
+    chunks() {
+      const res = [];
+      let left = 0;
+      for (let i = 0; i < this.sortedEntityPositions.length; i++) {
+        const e = this.sortedEntityPositions[i];
+        const l = this.makeLabel(left, e.start_offset);
+        res.push(l);
+        res.push(e);
+        left = e.end_offset;
+      }
+      const l = this.makeLabel(left, this.text.length);
+      res.push(l);
+
+      return res;
+    },
+
+    chunksWithLabel() {
+      return this.chunks().filter(r => this.id2label[r.label]);
```

Comment:
This line produces the following warn:

```
[Vue warn]: Error in render: "TypeError: this.chunks is not a function"

found in

---> <Annotator>
       <SequenceLabeling> at static/js/sequence_labeling.vue
         <Root>
```

By updating the code as follows, the warn will not occur.

```
return this.chunks.filter(r => this.id2label[r.label]);
```

Reply:
Good catch. Fixed in af26d38.